src/plugins/deploy/azureDeployPlugin.ts

```diff
@@ -31,6 +31,10 @@ export class AzureDeployPlugin extends AzureBasePlugin {
           "resourceGroup": {
             usage: "Resource group for the service",
             shortcut: "g",
+          },
+          subscriptionId: {
+            usage: "Sets the Azure subscription ID",
+            shortcut: "i",
```

Comment:
I'd expect to see `"s"` as the shortcut here

Reply:
-s is already being used to set the stage during deployment so the next best letter I decided to go with was i

Reply:
sounds good